src/player.rs

```diff
@@ -392,7 +392,7 @@ impl Player {
     /// See [`get_track_list`](Self::get_track_list) and [`supports_track_lists`](Self::supports_track_lists)
     /// if you want to manually handle compatibility checks.
```

Comment:
Please reference the extra checking function in the documentation comment too.

Reply:
Sure!